legal-api/src/legal_api/models/business.py

```diff
@@ -414,6 +416,24 @@ def good_standing(self):
                 return date_cutoff.replace(tzinfo=pytz.UTC) > datetime.utcnow()
         return True
 
+    @property
+    def in_dissolution(self):
```

Comment:
Can you simplify this to one query?
For example,
``` python
      check = db.session(Batch, BatchProcessing).\
          filter(Batch.id == BatchProcessing.batch_id).\
          filter(Batch.batch_type == Batch.BatchType.INVOLUNTARY_DISSOLUTION.value).\
          filter(<MORE CONDITIONS>).\
          one_or_none()
```

Reply:
Thank you for the feedback, I'm going to simplify the query. 
Just out of curiosity, Is the batch_type check for the future? Since currently we have only one type in the batch_type Enum.

Reply:
Yes